src/main/java/core/basesyntax/ColorSuplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSuplier {
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
You don't need to create an instance of Random object every time you calling the getRandomColor method